kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/flights/FlightsProcessor.java

```diff
@@ -0,0 +1,34 @@
+package com.kodilla.good.patterns.flights;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FlightsProcessor {
+
+    private List<Flight> flights;
+
+    public FlightsProcessor(List<Flight> flights) {
+        this.flights = flights;
+    }
+
+    public String findFlightFrom(CityType city) {
+        return flights.stream()
+                .filter(flight -> flight.getFrom() == city)
+                .map(Flight::toString)
+                .collect(Collectors.joining("\n"));
+    }
+
+    public String findFlightTo(CityType city) {
+        return flights.stream()
+                .filter(flight -> flight.getTo() == city)
+                .map(Flight::toString)
+                .collect(Collectors.joining("\n"));
+    }
+
+    public String findFlightThrough(CityType from, CityType to, CityType through) {
```

Comment:
To jest takie nierealne. W zadaniu jest "znalezienie lotow przez inne miasto". Moim zdaniem metoda powinna mieć jeden parametr :) City through (chyba raczej via)